pkg/controller/gather_commands.go

```diff
@@ -505,3 +512,26 @@ func boolToConditionStatus(b bool) metav1.ConditionStatus {
 	}
 	return conditionStatus
 }
+
+// updateClusterOperatorLastReportTime updates the ClusterOperator's lastReportTime extension field
+func updateClusterOperatorLastReportTime(ctx context.Context, client configv1.ConfigV1Interface) error {
+	insightsCo, err := client.ClusterOperators().Get(ctx, "insights", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	reported := status.Reported{
+		LastReportTime: metav1.Time{Time: time.Now().UTC()},
+	}
+
+	data, err := json.Marshal(reported)
+	if err != nil {
+		return fmt.Errorf("unable to marshal status extension: %v", err)
+	}
+	insightsCo.Status.Extension.Raw = data
```

Comment:
This will override all data in `Status.Extension`. Please verify we're not writing anything else there besides `LastReportTime`